README.md

```diff
@@ -129,12 +129,17 @@ Donations attributed to Participants or Teams
 |`displayName` (string)|The donor's name|
 |`donorID` (string)|The unique ID of the Donation|
 |`message` (string)|A message from the donor|
+|`numDonations` (float)|(Team and Participant Top Donor only) The total number of donations made.|
 |`participantID` (integer)|(Team Participant Donations only) The ID of the Participant this Donation is associated with|
+|`sumAmount` (float)|(Team and Participant Top Donor only) The total amount of donations.|
 |`teamID` (integer)|(Team and Team-Participant Donations only) The ID of the Team this Donation is associated with|
 
 #### Endpoints
 
 |Path|Description|
 |---|---|
 |`/api/participants/{participantID}/donations`|An Array of Donors associated to the Participant (To find `{participantID}`, visit your Participant's Fundraising Page. Look for the `&participantID=` URL parameter.)|
+|`/api/participants/{participantID}/donations/top`|A single object of the top donor for a Participant record (To find `{participantID}`, visit your Participant's Fundraising Page. Look for the `&participantID=` URL parameter.)|
```

Comment:
@wesme1337 i'm wondering since we're grouping the donations, if it makes sense for this to actually be `/api/participants/123/donors` - and expose `/top` - along with other values (`/newest`, '/oldest') as other options for all of our endpoints... let's chat this AM? i'm submitting a related issue